### PR TITLE
Modify namspaced xsi:object attribute values

### PR DESCRIPTION
**Pull request by pignion**
(hack) modify certain xml attributes that were receivening MS-incompa…tible namespaces in their values

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -49,6 +49,8 @@ public function __construct($wsdl, array $options = null)
      */
     public function __doRequest($request, $location, $action, $version, $one_way = 0)
     {
+        $request = $this->removeNameSpacedXsiObjectAttributeValues($request);
```
Comment (aedmonds) [inline src/Client.php]: It might be nice to have this be configurable as part of the class constructor. Do you think that's worth adding?

Comment (pignion) [reply src/Client.php]: there is no XML passed to the class consructor

Comment (pignion) [reply src/Client.php]: oh, configurable. Yeah, problem is we're not calling the constructor

Comment (pignion) [reply src/Client.php]: ok @aedmonds , I went ahead and made the change...

**Review by aedmonds — APPROVED** [review]

**Pull request merged**